helloworld-mdb/pom.xml

```diff
@@ -147,6 +148,12 @@
             <artifactId>jakarta.annotation-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.10.0</version>

```

Comment:
IMHO There is no value in having to manage Junit Jupiter version on each QS *now*, I think it would be preferable to now use standard junit like specified by the doc, which version comes managed from the EE with Tools BOM. I think it's better to re-evaluate Junit5 later as I told you, and if we go for such change we then add it's version management in the EE With Tools BOM. Still, if you want to proceed with Junit5 now you should define it's version in the properties like this:

<!-- The versions for other Dependencies and Plugins -->
<version.junit-jupiter-engine>5.10.0</version.junit-jupiter-engine>

... and then refer such property in dependencyManagement or directly in the dependency.

Reply:
I agree with @jamezp we should move to JUnit 5